<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regular_add</class>
 <widget class="QDialog" name="regular_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>정기 주차 등록</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="plate_txt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="plate_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>차량번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="start_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>정기주차 신청날짜</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>670</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>정기주차 종료날짜</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="start_date">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_txt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_txt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>670</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="register_btn">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>670</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="end_date">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>568</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>나가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>670</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>전화번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_txt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
